### PR TITLE
feat(dapp): pew link submit

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: packages/components/src/TCRUserDashboard/DashboardNewsroom.tsx [hunk 1]
```diff
@@ -153,6 +164,22 @@ const DashboardNewsroomBase: React.FunctionComponent<DashboardNewsroomProps> = p
...
 
+      <FeatureFlag feature={"pew"}>
+        <Query query={CHANNEL_ID_FROM_NEWSROOM_ADDRESS_QUERY} variables={{ contractAddress: props.newsroomAddress }}>
+          {({ loading, error, data }) => {
+            if (loading || error) {
+              return <></>;
+            }
+            return (
+              <StyledDashboardNewsroomSection>
+                <StyledDashboardNewsroomHdr>Pulse Story Feed</StyledDashboardNewsroomHdr>
```
Comment (sruddy) [inline packages/components/src/TCRUserDashboard/DashboardNewsroom.tsx]: Reminder to update when we have name and explainer text.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.17 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**